File: api/src/main/java/com/ruby/rubia_server/core/dto/UserDTO.java
package com.ruby.rubia_server.core.dto;

import com.ruby.rubia_server.core.enums.UserRole;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserDTO {
    
    private UUID id;
    private String name;
    private String email;
    private UUID companyId;
    private UUID departmentId;
    private String departmentName;
    private UserRole role;
    private String avatarUrl;
    private Boolean isOnline;
    private LocalDateTime lastSeen;
    private LocalDate birthDate;
    private Double weight;
    private Double height;
    private String address;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: api-dotnet/Rubia.Server/DTOs/UserDto.cs
using Rubia.Server.Enums;

namespace Rubia.Server.DTOs;

public class UserDto
{
    public Guid Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public Guid CompanyId { get; set; }
    public Guid DepartmentId { get; set; }
    public string? DepartmentName { get; set; }
    public UserRole Role { get; set; }
    public string? AvatarUrl { get; set; }
    public bool IsOnline { get; set; }
    public DateTime? LastSeen { get; set; }
    public DateOnly? BirthDate { get; set; }
    public double? Weight { get; set; }
    public double? Height { get; set; }
    public string? Address { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}